src/applications/debt-letters/components/OnThisPageLinks.jsx

```diff
@@ -6,7 +6,11 @@ const OnThisPageLinks = ({ isDetailsPage, hasHistory }) => (
     <h2>On this page</h2>
     <div className="vads-u-font-family--sans vads-u-display--flex vads-u-flex-direction--column">
       {!isDetailsPage && (
-        <a href="#currentDebts" className="vads-u-margin-y--2">
+        <a
+          href="#currentDebts"
+          className="vads-u-margin-y--2"
+          data-testid="debts-jumplink"
+        >
           <i
             aria-hidden="true"
             role="img"
```

Comment:
inconsequential but you can remove `role="img"` from these icons since they're set to `aria-hidden="true"`

Reply:
Done.